src/components/personalinformation/PersonalInformation.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const PersonalInformation = () => {
+  return <></>;
```

Comment:
an empty component?

Reply:
This is the component made from the design side.Will be implemented today.